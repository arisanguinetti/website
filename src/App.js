import React, { Component } from "react";
import {
  Contact,
  Footer,
  Hero,
  Navbar,
  Profile,
  Strengths
} from "./components";
import moment from "moment";

class App extends Component {
  constructor(props) {
    super(props);
    this.state = {
      birthdate: moment("1986-06-10")
    };
  }
  render() {
    const age = moment().diff(this.state.birthdate, "years");
    return (
      <React.Fragment>
        <Hero age={age} />
        <Navbar />
        <Profile age={age} />
        <Strengths />
        <Contact />
        <Footer />
      </React.Fragment>
    );
  }
}

export default App;
