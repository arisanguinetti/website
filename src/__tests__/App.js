import React from "react";
import { shallow } from "enzyme";
import App from "../App";

describe("App", () => {
  describe("Rendering", () => {
    it("should match to snapshot", () => {
      process.env.REACT_APP_ARIEL_BIRTHDAY = "2018-02-02";
      const component = shallow(<App />);
      expect(component).toMatchSnapshot();
    });
  });
});
