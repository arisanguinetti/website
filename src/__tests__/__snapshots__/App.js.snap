// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Rendering should match to snapshot 1`] = `
<Fragment>
  <Hero
    age={32}
  />
  <Navbar />
  <Profile
    age={32}
  />
  <Strengths />
  <Contact />
  <Footer />
</Fragment>
`;
